spec/requests/user_pages_spec.rb

```diff
@@ -11,5 +11,48 @@
       it { should have_content('Sign up') }
       it { should have_title(full_title('Sign up')) }
     end
+    describe "profile page" do
+      # Replace with code to make a user variable
+  
+      let(:user) { FactoryGirl.create(:user) }
```

Comment:
[IMO] As we include `FactoryGirl::Syntax::Methods`, we dont need `FactoryGirl`. `create` only is ok